x-pack/plugin/logsdb/src/yamlRestTest/resources/rest-api-spec/test/patternedtext/20_synthetic_source.yml

```diff
@@ -1,4 +1,8 @@
 simple:
+  - requires:
+      cluster_features: [ "mapper.patterned_text" ]
+      reason: "patterned_text mappings are used in this test"
+
```

Comment:
Ah, this isn't a `setup:` block. So you'll still need it for your second test in this file (`synthetic_source with copy_to`), or you'll need to add a setup block to this file and move the check there.